tests/src-android/cgeo/geocaching/storage/DataStoreTest.java

```diff
@@ -178,6 +181,22 @@ public static void testLoadLogsFromEmptyGeocode() {
         assertThat(logs).as("Logs for empty geocode").isEmpty();
     }
 
+    public static void testLog() {
+        //ensure that we don't overwrite anything in database before starting this test
+        DataStore.removeCache(ARTIFICIAL_GEOCODE, REMOVE_ALL);
+        assertThat(DataStore.loadLogs(ARTIFICIAL_GEOCODE)).isEmpty();
+
+        //TODO: create some logs, try to reload them
```

Comment:
Seems like it was done, is `todo` still valid?

Reply:
correct, will be removed